### PR TITLE
Add t0 chi2 to comparefitsreport

### PR DESCRIPTION
**Pull request by RoyStegeman**
As discussed during the meeting today, we should have the t0 chi2 info by default included in the fitreports.

### CODE REVIEW

File: validphys2/src/validphys/comparefittemplates/report.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ We are comparing:
 
 {@ summarise_fits @}
 
+{@ dataspecs::t0_info t0_chi2_info_table @}
```
Comment (Zaharid) [inline validphys2/src/validphys/comparefittemplates/report.md:14]: We might want to have a heading here.

Comment (RoyStegeman) [reply validphys2/src/validphys/comparefittemplates/report.md:14]: Now it's under the heading 'summary' where I would say it should be, I think a subsection under summary would be excessive. Do you disagree?

Comment (Zaharid) [reply validphys2/src/validphys/comparefittemplates/report.md:14]: Fair enough. The entries themselves say that is the t0.

Comment (RoyStegeman) [reply validphys2/src/validphys/comparefittemplates/report.md:14]: Exactly, I just made them separate tables since they can rely on different t0pdfsets

**Comment by scarlehoff** [comment]
I'm going to be picky and say that it should be either part of the summary table or completely separated, with its own heading.

**Comment by RoyStegeman** [comment]
A single table would of course be the obvious way to format it, but since the t0 settings would be different between entries reportengine cannot take care of it. So either we do this, or we just calculate the different t0 settings inside one of the summary functions out of reach from validphys (but this seems pretty ugly/bad practice).

But sine you're not alone with this opinion https://github.com/NNPDF/nnpdf/pull/1613#discussion_r990178548 I guess I'l place another heading.

**Pull request merged**